src/UI/Cells/NzbDroneCell.js

```diff
@@ -18,6 +19,25 @@ module.exports = Backgrid.Cell.extend({
                 }
             });
         }
+
+        this.render = _.wrap(this.render, this._renderDecorator);
+    },
+
+    _renderDecorator : function(renderInner) {
+        var el = this.$el;
+
+        if (this.$el.hasClass('editable')) {
```

Comment:
I think it would be better to pass a property for the cell that renders the cell and adds the class instead of relying on the DOM to control the flow.